expr/func_aspercent.go

```diff
@@ -60,8 +60,8 @@ func (s *FuncAsPercent) Exec(cache map[Req][]models.Series) ([]models.Series, er
 	}
 
 	if s.nodes != nil {
-		// List of keys
-		var keys []string
+		// List of keys (implemented as map for speed improvement)
```

Comment:
I'd probably ditch the stuff in parens. `map[string]struct{}` is golangs version of a set, so I'd just say "Set of keys"

Reply:
ok